<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1061</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="kach_kach">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>75</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>HSE Sans Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>графический отчет «кластеризованная столбчатая диаграмма» для пары «качественный атрибут —качественный атрибут»</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="kol_hach">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>75</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>HSE Sans Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>графический отчет «категоризированная гистограмма» для пары «количественный атрибут—качественный атрибут»</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="baw_kol_kach">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>75</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>HSE Sans Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>графический отчет «категоризированная диаграмма “box-and- whiskers”» для пары «количественный атрибут—качественный атрибут»</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="twokol_kach">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>75</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>HSE Sans Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>графический отчет «категоризированная диаграмма рассеивания» для двух количественных атрибутов и одного качественного атрибута</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
